frontend/index.html

```diff
@@ -531,15 +530,61 @@
         // and without this change, even if we set the dateVal to ''
         // we would try to load ntrips_purpose__default.png
         const imgFile = "plots/" + metric + "_" + dateVal + program + ".png";
+        const htmlFile = "plots/" + metric + "_" + dateVal + program + ".html";
         const altTextFile = "plots/" + metric + "_" + dateVal + program + ".txt";
         const altText = loadFile(altTextFile);
-        const nw = ["<li data-min-sizex='2' data-min-sizey='2'>"
+        console.log(metric)
+        console.log(`ntrips_${mode_studied}_total`)
```

Comment:
nit: do we need to keep these log statements?

Reply:
These log statements are not required, removed.